lib/rules/no-mixed-spaces-and-tabs.js

```diff
@@ -99,7 +99,7 @@ module.exports = {
 					 * At least one space followed by a tab
 					 * before non-tab/-space characters begin.
 					 */
-					regex = /^(?=(\t*))\1(?=( +))\2\t/u;
+					regex = /^\t* +\t/u;
```

Comment:
This is a common pattern to avoid backtracking. What was the reported problem in the original regex?

Reply:
That looks like a false positive in the `regexp/no-empty-lookarounds-assertion`. Since the lookahead has a capturing group, it isn't a no-op.